android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java

```diff
@@ -1413,8 +1409,7 @@ private void initializeBorder(KrollDict d, Integer bgColor)
 					if (radiusDim != null) {
 						radius = (float) radiusDim.getPixels(getNativeView());
 					}
-					if (radius > 0f && HONEYCOMB_OR_GREATER &&
-							(LOWER_THAN_JELLYBEAN || (d.containsKey(TiC.PROPERTY_OPACITY) && LOWER_THAN_MARSHMALLOW))) {
+					if (radius > 0f && HONEYCOMB_OR_GREATER && d.containsKey(TiC.PROPERTY_OPACITY)) {
```

Comment:
This change needs to be reverted, the circle in this test case will become a square on Jelly Bean 4.2 and below.

```JS
var window = Ti.UI.createWindow(),
    webView = Ti.UI.createWebView({
	    url: 'https://www.quirksmode.org/html5/tests/video.html',
	    borderWidth: '20',
	    borderColor: 'red',
	    width: '66%', height: '66%',
	    borderRadius: '20'
    }),
    circle = Ti.UI.createView({
        top: 10,
        backgroundColor: 'blue',
        width: 50, height: 50,
        borderRadius: 25
    });
window.add([webView, circle]);
window.open();
```